src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/reporting/generator/BookingsReportGenerator.kt

```diff
@@ -1,16 +1,29 @@
 package uk.gov.justice.digital.hmpps.approvedpremisesapi.reporting.generator
 
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BookingEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.TemporaryAccommodationApplicationEntity
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.reporting.model.BookingsReportRow
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.reporting.properties.BookingsReportProperties
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.reporting.util.toShortBase58
 import java.time.LocalDate
 import java.time.temporal.ChronoUnit
 
 class BookingsReportGenerator : ReportGenerator<BookingEntity, BookingsReportRow, BookingsReportProperties>(BookingsReportRow::class) {
 
   override val convert: BookingEntity.(properties: BookingsReportProperties) -> List<BookingsReportRow> = {
+    val application = this.application as? TemporaryAccommodationApplicationEntity
+
     listOf(
       BookingsReportRow(
+        referralId = application?.id?.toShortBase58(),
```

Comment:
I'd assumed that we just wanted to present an arbitrary unique reference here similar to what we do in the bed usage/utilisation reports that can be used to quickly refer to a particular application by the users beyond the boundaries of the service (such as email or Slack messages), rather than this being a potential source of support requests. I think UUIDs aren't especially human readable as they're pretty long and if you're not familiar with them have characters that can be easily confused with invalid ones (**0** and **O** for example). Regardless, I've changed this to a UUID.